### DISCUSSION TITLE
How do you create a redistribuable Symfony UX package ?

### DISCUSSION THREAD

**Discussion by tjamps** [symfony/symfony#49483] [Q&A]
I'm totally impressed by the Symfony UX initiative, I find it really great :heart_eyes: 

I would like to be able to create a Symfony UX package so that I can add it to my composer.json, just like you would do with the packages featured on [ux.symfony.com](https://ux.symfony.com/)

I did not find any directions on this subject, did I overlook something in the documentation ?

I would gladly enhance the documentation on the subject if 1) it is missing 2) I have some directions on how to accomplish that :slightly_smiling_face:

**Comment by stof** [comment]
There was a discussion a few days ago on Slack about the fact that this documentation is currently missing. You should contact @weaverryan about that topic if you want to help on that doc (and you should probably discuss that in the `#ux` channel on the Symfony slack)

**Comment by weaverryan** [comment]
Here is the PR for the docs - https://github.com/symfony/symfony-docs/pull/17930 - it would be great if you could try it and see if anything is confusing or missing :).
